packages/salesforcedx-vscode-core/src/commands/forceApexLogGet.ts

```diff
@@ -208,14 +214,60 @@ export class ForceApexLogList {
   }
 }
 
-const workspaceChecker = new SfdxWorkspaceChecker();
-const parameterGatherer = new LogFileSelector();
+export class ApexLibraryGetLogsExecutor extends ApexLibraryExecutor {
+  protected logService: LogService | undefined;
+
+  public createService(conn: Connection): void {
+    this.logService = new LogService(conn);
+  }
+
+  public async execute(
+    response: ContinueResponse<{ id: string }>
+  ): Promise<void> {
+    try {
+      await this.build(
+        nls.localize('apex_log_get_text'),
+        nls.localize('force_apex_log_get_library')
+      );
+
+      if (this.logService === undefined) {
+        throw new Error('Log Service is not established.');
+      }
+
+      this.logService.getLogs = this.getLogsWrapper(this.logService.getLogs);
+      const id = response.data.id;
+      const logDir = path.join(
+        getRootWorkspacePath(),
+        '.sfdx',
+        'tools',
+        'debug',
+        'logs'
+      );
+      await this.logService.getLogs({
+        logId: id,
+        outputDir: logDir
+      });
+    } catch (e) {
+      telemetryService.sendException(
+        nls.localize('force_apex_log_get_library'),
+        e.message
+      );
+      notificationService.showFailedExecution(this.executionName!);
```

Comment:
Nit: we don't need the `!` 
```suggestion
      notificationService.showFailedExecution(this.executionName);
```